### PR TITLE
Removes un-used files

### PR DESCRIPTION
**Pull request by simonmitchell**

### CODE REVIEW

File: ThunderBasics/JSONSerialization+FromFile.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//
+//  Created by Simon Mitchell on 06/02/2018.
+//  Copyright © 2018 threesidedcube. All rights reserved.
+//
+
+import Foundation
+
+extension JSONSerialization {
+	
+	open class func jsonObject(with resource: String?, extension fileExtension: String?, in bundle: Bundle = .main, options opt: JSONSerialization.ReadingOptions = []) throws -> Any {
```
Comment (MattCheetham) [inline ThunderBasics/JSONSerialization+FromFile.swift]: Can you document this file please?

File: ThunderBasics/MapView+Fitting.swift [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+//  ThunderBasics
+//
+//  Created by Simon Mitchell on 06/02/2018.
+//  Copyright © 2018 threesidedcube. All rights reserved.
+//
+
+import Foundation
+import MapKit
+
+public extension MKMapView {
```
Comment (MattCheetham) [inline ThunderBasics/MapView+Fitting.swift:12]: Please document this extension and methods

**Review by MattCheetham — CHANGES_REQUESTED** [review]
Looks good. Goodbye 7000 lines! Just document the new bits and we're good.

**Comment by simonmitchell** [comment]
@MattCheetham documented! Tried to copy a lot of the documentation from similar methods on the Apple classes so they should be fairly explanatory 😊

**Review by ryanbourneuk — APPROVED** [review]

**Pull request merged**